schema/bom-1.7.xsd

```diff
@@ -554,12 +554,24 @@ limitations under the License.
                         of the component. Examples: commons-lang3 and jquery</xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="version" type="bom:versionType" minOccurs="0" maxOccurs="1">
-                <xs:annotation>
-                    <xs:documentation>The component version. The version should ideally comply with semantic versioning
-                        but is not enforced.</xs:documentation>
-                </xs:annotation>
-            </xs:element>
+            <xs:choice>
+                <xs:element name="version" type="bom:versionType" minOccurs="0" maxOccurs="1">
```

Comment:
Shouldn't `minOccurs` and `maxOccurs` be specified on the `<xs:choice>` element instead?